code/index.html

```diff
@@ -1,22 +1,74 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <title>Travel magazine</title>
+</head>
+<body>
+  <header class="header">
+    <div>
+      <img class="logo" src="logo.png" alt="logo">
+    </div>
+    <div class="nav">
+        <nav class="nav-elements">
+          <a class="nav-el" href="">HOME</a>
+          <a class="nav-el" href="">DESTINATIONS</a>
+          <a class="nav-el" href="">CULTURE</a>
+          <a class="nav-el" href="">TRICKS & TIPS</a>
+        </nav>
+        <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+    </div>
+  </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+  <section class="big-news">
+    <img src="images/camper.jpg" alt="camper">
+    <div class="big-news-text">
+    <h1>2020 was the year of the campervan</h1>
+    <p>2020 was the year of the campervan – but could 2021 be even busier, as people now see them as 'the safe way to travel'?</p>
+    <button class="btn" type="button">Read more</button>
+  </div>
+  </section>
```

Comment:
Indent for every nested element:
```
<section class="big-news">
  <img src="images/camper.jpg" alt="camper">
  <div class="big-news-text">
    <h1>2020 was the year of the campervan</h1>
    <p>2020 was the year of the campervan – but could 2021 be even busier, as people now see them as 'the safe way to travel'?</p>
    <button class="btn" type="button">Read more</button>
  </div>
</section>
```

Reply:
Friendly reminder that you still have some indentation that needs to be fixed :)